sdk/identity/azure-identity/CMakeLists.txt

```diff
@@ -38,9 +38,9 @@ if(FETCH_SOURCE_DEPS)
         add_subdirectory(${oneFolder} EXCLUDE_FROM_ALL)
     endforeach()
 elseif(NOT AZ_ALL_LIBRARIES)
-  find_package(azure-core-cpp "1.2.0" CONFIG QUIET)
+  find_package(azure-core-cpp "1.3.1" CONFIG QUIET)
```

Comment:
This seems reasonable. However, can you help me understand why this should be 1.3.1 (i.e. what we have GA'd in main vcpkg repo) instead of latest beta?